File: app/src/main/java/com/lcl6/cn/basedialog/mvp/model/NewsModel.java
package com.lcl6.cn.basedialog.mvp.model;

import com.lcl6.cn.basedialog.bean.JsoupBean;

import java.util.List;

/**
 * Created by liancl on 2017/8/23.
 */

public interface NewsModel {

    void getData(LoadCompleteListener listener);

    interface  LoadCompleteListener{
        void comlpete(List<JsoupBean> list);
    }


}

File: app/src/main/java/com/lcl6/cn/basedialog/mvp/model/NewsModel.kt
package com.lcl6.cn.basedialog.mvp.model

import com.lcl6.cn.basedialog.bean.JsoupBean

/**
 * Created by liancl on 2017/8/23.
 */

interface NewsModel {

    fun getData(listener: LoadCompleteListener)

    interface LoadCompleteListener {
        fun comlpete(list: List<JsoupBean>)
    }


}
